cmd/minikube/cmd/start.go

```diff
@@ -691,3 +693,12 @@ func saveConfig(clusterConfig cfg.Config) error {
 	}
 	return nil
 }
+
+func determineLocale() {
```

Comment:
This should be in `main.go`, so that it affects all commands, rather than just `minikube start`.

Reply:
Makes sense. done.